community/code-of-conduct.md

```diff
@@ -0,0 +1,166 @@
+10. Cat calls.
+11. Sexual comments.
+12. Turning work discussions to sexual topics.
+13. Sexual innuendos or stories.
+14. Asking about sexual fantasies, preferences, or history.
+15. Personal questions about social or sexual life.
+16. Sexual comments about a person's clothing, anatomy, or looks.
+17. Kissing sounds, howling, and smacking lips.
+18. Telling lies or spreading rumors about a person's personal sex life.
+19. Neck massage.
+20. Touching an employee's clothing, hair, or body.
+21. Giving personal gifts., Hanging around a person.
+22. Hugging, kissing, patting, or stroking.
+23. Touching or rubbing oneself sexually around another person.
+24. Standing close or brushing up against a person.
+25. Looking a person up and down (elevator eyes).
+26. Staring at someone.
+27. Sexually suggestive signals.
+28. Facial expressions, winking, throwing kisses, or licking lips.
+29. Making sexual gestures with hands or through body movements.
+
+
+
+The point of having this list, is to make people aware that these things can be considered offensive and harassment, by some people. We come from different culture, for example hugging might be quite common in a few culture, but might not be well received by some other culture. So asking “if a person is comfortable” might help in all verbal/non-verbal/physical contact.
+
+## What is considered as bullying ?
+
+[Ref](https://www.stopbullying.gov/bullying/what-is-bullying)
+
+1. Verbal bullying, is saying or writing mean things. Verbal bullying includes:
+   1. Teasing
+   2. Name-calling
+   3. Inappropriate sexual comments
+   4. Taunting
+   5. Threatening to cause harm
+
+1. Social bullying, sometimes referred to as relational bullying, involves hurting someone’s reputation or relationships. Social bullying includes:
+
+   1. Leaving someone out on purpose
+   2. Telling other children not to be friends with someone
+   3. Spreading rumors about someone
+   4. Embarrassing someone in public
+
+1. Physical bullying , involves hurting a person’s body or possessions. Physical bullying includes:
+
+   1. Hitting/kicking/pinching
+   2. Spitting
+   3. Tripping/pushing
+   4. Taking or breaking someone’s things
+   5. Making mean or rude hand gestures
+
+## Basic Ownership Model
+
+Subject -> Who has been accused of harassment/bullying
+
+The actor owns the action and it's consequence like hurting or acting as disabler. You shouldn't blame the affected person for being affected.
+
+Having empathy and good will for the affected person is great, but the truth is :-
+
+1. Just asking/demanding for a thicker skin won't magically generate a thicker skin.
+2. Not every time humans have the energy and need to change.
+
+So rather than expecting and introducing toxic culture for developing thicker skin, the subject is requested to just fucking change themselves temporarily/selectively. It might be general or it can be personalized, but change is necessary.
+
+## Status Game
+
+Try not to play status game by :-
```

Comment:
Can also be termed as gaslighting.

Reply:
I don't think so. Playing status game is a power move, gaslighting is more of making someone believe something about themselves or events or memories that isn't true

Reply:
True, we should cover both cases then.